pom.xml

```diff
@@ -169,8 +175,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>17</source>
-                    <target>17</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
Suggest revert this change since our project is using the latest LTS version (17) of JAVA.